politeiawww/cmd/piwww/commentvotes.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2017-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+
+	pi "github.com/decred/politeia/politeiawww/api/pi/v1"
+	"github.com/decred/politeia/politeiawww/cmd/shared"
+)
+
+// CommentVotesCmd retreives like comment objects for
+// the specified proposal from the provided user.
+type CommentVotesCmd struct {
+	Args struct {
+		Token  string `positional-arg-name:"token"`  // Censorship token
+		UserID string `positional-arg-name:"userid"` // User id
+	} `positional-args:"true" required:"true"`
+
+	// CLI flags
+	Vetted   bool `long:"vetted" optional:"true"`
+	Unvetted bool `long:"unvetted" optional:"true"`
```

Comment:
Since we don't allow unvetted votes anymore, we can just assume this is vetted.  No need for the CLI flags.  Same for the `CommentVote` command.